data/obs-packaging/README.rst

```diff
@@ -5,31 +5,33 @@ Intel® Clear Containers OBS specification files
 ###############################################
 
 
-The `Clear Containers`_ packages are available via `Open Build Service`_
+The `Intel® Clear Containers`_ packages are available via `Open Build Service`_
```

Comment:
The addition is not needed, the correct trademark was used already on the title.